backend/src/main/java/disaster/service/disaster_alert/DisasterAlertHazardEventsProvider.java

```diff
@@ -0,0 +1,24 @@
+package disaster.service.disaster_alert;
```

Comment:
not sure that's a good idea to use underscore for the package name
https://stackoverflow.com/questions/49890803/naming-conventions-of-composed-package-names